fe/fe-core/src/main/java/org/apache/doris/analysis/ExportStmt.java

```diff
@@ -244,7 +244,7 @@ public static String checkPath(String path, StorageBackend.StorageType type) thr
             }
         } else if (type == StorageBackend.StorageType.S3) {
             if (schema == null || !schema.equalsIgnoreCase("s3")) {
-                throw new AnalysisException("Invalid export path. please use valid 'S3://' path.");
+                throw new AnalysisException("Invalid export path. please use valid 's3://' path.");
```

Comment:
!schema.equalsIgnoreCase("s3")

why we need change 'S3' to 's3'?

Reply:
When the s3 path entered by the user is wrong, we should return the correct prompt information, and returning 'S3' is wrong, so I changed to the correct 's3'